src/renderer/src/components/settings/Setting.tsx

```
@@ -24,15 +27,24 @@ const Settings = (): ReactElement => {
     selectSettingByEmail(state, currentUser),
   );
 
+  const [selectedLanguage, setSelectedLanguage] = useState(i18n.language);
+
   useEffect(() => {
     if (currentUser) {
       dispatch(fetchSettings({ email: currentUser }));
     }
   }, [dispatch, currentUser]);
 
+  const handleLanguageChange = (_: any, value: string | null) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)